src/apis/authApi.js

```diff
@@ -0,0 +1,13 @@
+import axios from './axiosClient';
+
+export const resetPassword = async ({ token, password }) => {
+  try {
+    const response = await axios.post('/api/v1/auth/forgot_password', {
```

Comment:
Our `axiosClient` already adds `/api/v1` to any routes we call so that we don't have to add that to every http call ourselves
https://github.com/Code-the-Dream-School/dd-prac-team4-front/blob/page_update_password/src/apis/axiosClient.js#L3C29-L3C52

(it uses `REACT_APP_API_BASE_PATH` as the base path for any http call made using this base axios client which is defined in our env file as `http://localhost:8000/api/v1` for local dev and similarly for the deployed one: https://github.com/Code-the-Dream-School/dd-prac-team4-front/blob/page_update_password/.env#L1)

Otherwise, right now the code you have here will try to make a call to `http://localhost:8000/api/v1/api/v1/auth/forgot_password`

Reply:
Corrected 👍, commit 1f6dc4e98f593f7aa8e0ad1fde3e767fbcca940d